modelbuilders/bench.py

```diff
@@ -0,0 +1,509 @@
+import argparse
+import numpy as np
+import sklearn
+import timeit
+import json
+
+
+def columnwise_score(y, yp, score_func):
```

Comment:
bench.py file should be similar in all folder (sklearn, daal4py, etc.)

Reply:
Thank you. Added same `bench.py` as in other folders.
I also added the `utils.py` file with function, that I use in both my benchmarks